package.json

```diff
@@ -22,7 +22,8 @@
     "serve": "node ./dev-utils/run-script.js startTestServers",
     "coverage": "lerna run build:module && lerna run karma:coverage --stream",
     "bench:dev": "lerna run build:module && node ./scripts/benchmarks.js",
-    "clean": "npx lerna exec -- rm -rf dist/"
+    "clean": "npx lerna exec -- rm -rf dist/",
+    "release-ci": "lerna publish --otp=${TOTP_CODE} --yes && npm run github-release"
```

Comment:
This is the release process to be used within the CI:
- Skip all confirmation prompts with the `--yes` flag
- Pass the OTP token with the `--otp` flag.

Reply:
We can remove `--yes` here as well.